portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,17 +49,16 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    System.out.println(request.getParameter("lang"));
-
     List<String> comments = new ArrayList<>();
     Query query = new Query(ENTITY_KIND).addSort(ENTITY_TIMESTAMP_HEADER, SortDirection.ASCENDING);
     PreparedQuery results = datastore.prepare(query);
-
+    String prevLanguageCode = request.getParameter("sourceLanguageCode");
+    String languageCode = request.getParameter("targetLanguageCode");
     for (Entity entity : results.asIterable()) {
-      comments.add(buildComment(entity));
+      comments.add(buildComment(entity, prevLanguageCode, "languageCode"));
```

Comment:
You should use languageCode here? Not "languageCode"

Reply:
Done, my bad